src/components/ModuleFields/Configurator/File.vue

```diff
@@ -45,6 +45,138 @@
+          <b-input-group
+            :append="$t('kind.file.view.px')"
+          >
+            <b-form-input
+              v-model="f.options.height"
+              type="number"
+              :placeholder="$t('kind.file.view.height')"
+            />
+          </b-input-group>
+        </b-form-group>
+      </b-col>
+      <b-col
+        cols="6"
+        align-self="center"
+      >
+        <b-form-group>
+          <label>{{ $t('kind.file.view.width') }}</label>
+          <b-input-group
+            :append="$t('kind.file.view.px')"
+          >
+            <b-form-input
+              v-model="f.options.width"
+              type="number"
+              :placeholder="$t('kind.file.view.width')"
+            />
+          </b-input-group>
+        </b-form-group>
+      </b-col>
+
+      <b-col
+        cols="6"
+        align-self="center"
+      >
+        <b-form-group>
+          <label>{{ $t('kind.file.view.maxHeight') }}</label>
+          <b-input-group
+            :append="$t('kind.file.view.px')"
+          >
+            <b-form-input
+              v-model="f.options.maxHeight"
+              type="number"
+              :placeholder="$t('kind.file.view.maxHeight')"
+            />
+          </b-input-group>
+        </b-form-group>
+      </b-col>
+      <b-col
+        cols="6"
+        align-self="center"
+      >
+        <b-form-group>
+          <label>{{ $t('kind.file.view.maxWidth') }}</label>
+          <b-input-group
+            :append="$t('kind.file.view.px')"
+          >
+            <b-form-input
+              v-model="f.options.maxWidth"
+              type="number"
+              :placeholder="$t('kind.file.view.maxWidth')"
+            />
+          </b-input-group>
+        </b-form-group>
+      </b-col>
+
+      <b-col
+        cols="6"
+        align-self="center"
+      >
+        <b-form-group>
+          <label>{{ $t('kind.file.view.borderRadius') }}</label>
+          <b-input-group
+            :append="$t('kind.file.view.px')"
+          >
+            <b-form-input
+              v-model="f.options.borderRadius"
+              type="number"
+              :placeholder="$t('kind.file.view.borderRadius')"
+            />
+          </b-input-group>
+        </b-form-group>
+      </b-col>
+      <b-col
+        cols="6"
+        align-self="center"
+      >
+        <b-form-group
+          :label="$t('kind.file.view.background')"
+        >
+          <b-form-input
+            v-model="f.options.backgroundColor"
+            type="color"
+            debounce="300"
+            class="mb-1"
+          />
+        </b-form-group>
+      </b-col>
+
+      <b-col
+        cols="6"
+        align-self="center"
+      >
+        <b-form-group>
+          <label>{{ $t('kind.file.view.margin') }}</label>
+          <b-input-group
+            :append="$t('kind.file.view.px')"
+          >
+            <b-form-input
+              v-model="f.options.margin"
+              type="number"
```

Comment:
If you do change the width, height and so on values to number type. Then make sure to also use the number prop here (look at bootstrap input number prop)